code/Control/UserDefinedFormController.php

```diff
@@ -104,10 +104,13 @@ protected function addUserFormsValidatei18n()
      * where the form should be rendered into. If it does not exist
      * then default back to $Form.
      *
-     * @return array
+     * @return array|Object
      */
     public function index(HTTPRequest $request = null)
     {
+        if ($this->config()->disable_form_content_interpolation) {
+            return $this;
+        }
```

Comment:
In any which way the BC conversation goes - the 'default' return type on line 131 makes it moot (although it itself is completely redundant - it is pre-existing).
Ideally the config check as the conditional for this if statement should simply be negated & an extra and clause on the one below.

Reply:
Good point.
If we change change it too:
```
if ($this->Content && $form = $this->Form() && !$this->config()->disable_form_content_interpolation) {
```
The return will stay as an array and the `$Form` will still be available to templates.